### ISSUE TITLE
Pivot tables - sometimes pivot tables are not shown correctly while editing dashboard

### ISSUE THREAD

**Issue by mtwestra** [akvo/akvo-lumen#623]
### Context
How to reproduce:
1) create a dashboard with a pivot table on it
2) edit the dashboard for editing
=> pivot table is shown as empty square with 'Please choose a dataset', as shown below.

<img width="379" alt="screen shot 2017-02-28 at 14 49 33" src="https://cloud.githubusercontent.com/assets/478954/23407991/a977411e-fdc6-11e6-828b-782c495111ef.png">

### Expected behaviour
Pivot table should display correctly.

**Comment by gabemart** [comment]
This bug was caused by the visualisation-loading code for the dashboard container assuming every visualisation required loading a dataset.

Previously, to avoid loading the same dataset multiple times in the case of a dashboard that has multiple visualisations depending on that same dataset, we kept a record of requested datasets in the dashboard editor state. Once a dataset had been requested, we ignored visualisations depending on that dataset, trusting that the first request would complete and then dashboard would update, loading all the visualisations that depended on that dataset

Pivot tables require an API call to receive aggregated data, but nonetheless, the same code that ignored repeated requests for the same dataset was suppressing this call in cases where a pivot table was created from the same dataset as another visualisation type, and both visualisations were included in the same dataset.

To fix this, I've added a list of visualisation types that require a special call for aggregated data (even though this is only "pivot table" for the time being), and added a check against this list, so pivot tables will always make their API call when loading a dashboard in the editor
